.github/workflows/win.yml

```diff
@@ -284,14 +284,34 @@ jobs:
           .\crystal-cross.exe build src/compiler/crystal.cr -Di_know_what_im_doing -Dwithout_playground --link-flags=/STACK:10000000
           mv crystal.exe bin/
 
+      - name: Download latest shards release
+        uses: actions/checkout@v2
+        with:
+          repository: crystal-lang/shards
+          ref: v0.16.0
+          path: shards
+
+      - name: Download molinillo release
+        uses: actions/checkout@v2
+        with:
+          repository: crystal-lang/crystal-molinillo
+          ref: v0.2.0
+          path: shards\lib\molinillo
```

Comment:
We should stick with paths with forward slashes particularly in direct YAML args